fdbserver/ClusterController.actor.cpp

```diff
@@ -2682,6 +2686,40 @@ ACTOR Future<Void> workerHealthMonitor(ClusterControllerData* self) {
 	}
 }
 
+ACTOR Future<Void> metaclusterMetricsUpdater(ClusterControllerData* self) {
+	loop {
+		if (self->db.clusterType == ClusterType::METACLUSTER_MANAGEMENT) {
```

Comment:
We could structure this to wait(Never()) if it isn't a management cluster and then also wait for changes to the server DB info that would signal we may need to try again.